web/src/routes/(user)/trash/+page.svelte

```diff
@@ -52,8 +56,12 @@
     try {
       await restoreTrash();
 
+      const restoredAssetIds = assetStore.assets.map((a) => a.id);
+      const numberOfAssets = restoredAssetIds.length;
+      assetStore.removeAssets(restoredAssetIds);
```

Comment:
this is not correct for restoring, correct?

Reply:
I don't know for sure honestly. It seems different than other restore methods as it uses the sdk which I am still a bit unsure where to find. All I am doing is updating the assetStore which gives the visual update. Please correct me if I'm wrong, still learning the code base and svelte.

Reply:
oh wait, this is the assetStore of this current page, I think you are correct here